cssom-view-1/Overview.bs

```diff
@@ -1141,7 +1141,7 @@ aborting on the first step that returns a value:
     1. Return the {{DOMRect}} object in <var>list</var> at index 0.
 1. If <a>caret node</a> is a text entry widget that is a replaced element,
     and that is in the document,
-    return a {{DOMRect}} object for the caret in the widget
+    return an [=unscaled=] {{DOMRect}} object for the caret in the widget
```

Comment:
This seems like it should return scaled lengths too, isn't these global client coordinates like getBoundingClientRect()?

Reply:
Yes, that was a mistake. Fixed.